iana/entID.go

```diff
@@ -0,0 +1,4 @@
+package iana
```

Comment:
You created a file with the name `entID.go`.

All the rest of the file names are lower case:

```
archtype.go
hwtypes.go
iana.go
statuscodes.go
```

 Can it be lower case for consistency?

Reply:
sure will do